### ISSUE TITLE
Rename the instance folder when renaming instances

### ISSUE THREAD

**Issue by Aces-and-Jacks** [PrismLauncher/PrismLauncher#1840]
### Role

I play modded Minecraft

### Suggestion

Renaming an instance renames the folder

### Benefit

So that I won't get confused

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

So when renaming instances I would also need to rename the folder so I don't get confused. Luckily when clicking the "folder" section it goes to the instance but this would be useful. Don't know how hard it'd be tho

"Minor fluff"
I'm surprised no one has suggested this. It has been an issue in the past but I just ignored it. Then I was browsing multiMC and came by this. I realized that this would be very convenient so I figured it'd be cool to suggest.

**Comment by Aces-and-Jacks** [comment]
Tho looking at how they stopped trying, it might not be possible, since it breaks folder linking. I'll still keep it open and if someone else comes to the same conclusion they can close this

**Comment by ghost** [comment]
I renamed the instance in Prism, closed the launcher, and manually renamed the folder to the same thing. Zero issues and Prism can tell where the pack is still!

**Comment by Aces-and-Jacks** [comment]
Yeah, that's what I've always done. Well I guess it wouldn't cause issues for prism to rename. But I could be wrong

**Comment by Alyaqdhans** [comment]
Yes, we need this feature, my instance folder names are all messed up.

**Comment by raxod502** [comment]
Ok, so just in case anyone what's happening behind the scenes: it seems that there's no central database of instances in PrismLauncher, it just scans the `instances` directory on startup to see what instances exist. Each instance has a `.cfg` file inside its directory which contains the *display* name that is used inside the launcher.

So, to change the folder for an instance, just rename the folder in your filesystem editor, and restart PrismLauncher!

**Comment by Aces-and-Jacks** [comment]
Wow, didn't think this would ever be implemented. thank you Mick235711
